lpipe/pipeline.py

```diff
@@ -366,11 +364,9 @@ def return_handler(ret: Any, state: State) -> Any:
         state.logger.debug(f"executing dynamic payload: {p}")
         try:
             ret = execute_payload(payload=p, state=state)
-        except Exception as e:
-            state.logger.debug(utils.exception_to_str(e))
-            raise lpipe.exceptions.FailButContinue(
-                f"Failed to execute returned Payload: {p}"
-            ) from e
+        except Exception:
```

Comment:
The old version used FailButContinue, while this one raises explicitly; will this cause apps to break where they wouldn't before, when they trigger this?

Reply:
So, this was actually a bug. With this change, any exceptions raised by subsequent Payloads in return_handler will be subject to the same error handling code that all other errors have been. This may uncover issues where errors were being steamrolled, but considering this is a brand new feature, and I think Alpha is the first to really use it, I think the impact should be close to zero.

Reply:
So, this was actually a bug. With this change, any exceptions raised by subsequent Payloads in return_handler will be subject to the same error handling code that all other errors have been. This may uncover issues where errors were being steamrolled, but considering this is a brand new feature, and I think Alpha is the first to really use it, I think the impact should be close to zero.